pkg/api/validation/validation.go

```diff
@@ -1116,6 +1124,20 @@ func validateLocalVolumeSource(ls *api.LocalVolumeSource, fldPath *field.Path) f
 	return allErrs
 }
 
+func validateStorageOSVolumeSource(storageos *api.StorageOSVolumeSource, fldPath *field.Path) field.ErrorList {
```

Comment:
when validating a PV storageos volume source, need to make sure that the secret name and namespace are both specified, or neither is specified, right?